spec/lib/license_finder/package_managers/composer_package_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module LicenseFinder
+  describe ComposerPackage do
+    subject do
+      described_class.new('symfony/debug', 'v3.0.7', 'license' => [{ 'name' => 'MIT' }])
+    end
+
+    its(:name) { should == 'symfony/debug' }
+    its(:version) { should == 'v3.0.7' }
+    its(:summary) { should eq '' }
+    its(:description) { should == '' }
+    its(:homepage) { should == '' }
+    its(:package_manager) { should eq 'Composer' }
+
+    describe '#license_names_from_spec' do
+      it 'finds the license for both license structures' do
```

Comment:
I don't know much about composer but in #556 there were additional test cases here to handle different formats. Why was this removed?

Reply:
Those were all taken from NPM's test suite and not relevant to composer.